lib/taskana-core-test/src/test/java/acceptance/ArchitectureTest.java

```diff
@@ -392,6 +429,38 @@ void exceptionsShouldBePublic() {
 
   // region Helper Methods
 
+  /**
+   * Test the dependencies of the packages. Adds the prefix <code>pro.taskana</code> to every given
+   * value.
+   *
+   * @param module the module which should be tested
+   * @param dependentModules the expected dependent modules
+   */
+  private void moduleShouldOnlyDependOn(String module, List<String> dependentModules) {
+
+    String moduleTemplate = "pro.taskana.%s..";
+
+    String moduleUndertest = String.format(moduleTemplate, module);
+
+    List<String> dependentModulesList =
+        dependentModules.stream()
+            .map(dp -> String.format(moduleTemplate, dp))
+            .collect(toCollection(ArrayList::new));
```

Comment:
You can directly collect the stream to an array. Then you don't have to convert the list to an array a few lines below.